docs/docs/nlg.mdx

```diff
@@ -28,7 +28,7 @@ like this:
 
 ```json
 {
-  "template":"utter_what_can_do",
+  "response":"utter_what_can_do",
```

Comment:
@wochinge, shouldn't line 20 be Request format? It's the format of the request we're sending to the nlg server, no?

Reply:
Very good catch!